### PR TITLE
Implemented Settings.{modifyFacebookPages, removeFacebookPages}

### PR DESCRIPTION
**Pull request by jcjimenez**
modify_facebook_pages

### CODE REVIEW

File: src/resolvers-cassandra/Settings/mutations.js [hunk 1]
```diff
@@ -19,13 +20,64 @@ const TRUSTED_SOURCES_RANK_DEFAULT = 10;
...
  * @param {{input: {site: string, pages: Array<{pageUrl: string, RowKey: string}>}}} args
  * @returns {Promise.<{runTime: string, pages: Array<{pageUrl: string, RowKey: string}>}>}
  */
 function modifyFacebookPages(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
-    reject('Modification not yet supported.');
+    const pages = args && args.input && args.input.pages;
+    if (!pages || !pages.length) return reject('No pages specified');
+    
+    const invalidPages = pages.filter(pages=>!pages.pageUrl);
```
Comment (c-w) [inline src/resolvers-cassandra/Settings/mutations.js]: FYI: Unless you want to provide the nice error message, you can also use `pages.some(page => !page.pageUrl)`

Comment (c-w) [inline src/resolvers-cassandra/Settings/mutations.js]: Maybe better to not over-write the pages variable in the filter scope. `pages.filter(page => !page.pageUrl)`

**Review by c-w — APPROVED** [review]
LGTM.

File: src/resolvers-cassandra/Settings/mutations.js (hunk 1 above)
Comment (jcjimenez) [reply src/resolvers-cassandra/Settings/mutations.js]: Nice! Will try that.

Comment (jcjimenez) [reply src/resolvers-cassandra/Settings/mutations.js]: Good catch.

**Pull request merged**